docs/book/releases/v2.2.0.md

```diff
@@ -0,0 +1,72 @@
+<!-- markdownlint-disable MD034 -->
+# vSphere CSI Driver - v2.2.0 release
+
+## New Feature
+
+- Online Volume Expansion. Refer [Volume Expansion documentation](../features/volume_expansion.md) for detail.
+
+## Notable Changes
+
+- Datastore privilege check to filter out shared datastores which do not have `Datastore.FileManagement` privilege. Refer notes provided for `CNS-DATASTORE` role at [vSphere Roles and Privileges](https://vsphere-csi-driver.sigs.k8s.io/driver-deployment/prerequisites.html#vsphere-roles-and-privileges-)
+- Telemetry collection support using `cluster-distribution` field in the vSphere Config Secret. Refer notes provided for `cluster-distribution` in [vSphere configuration secret](https://vsphere-csi-driver.sigs.k8s.io/driver-deployment/installation.html#vsphere-configuration-file-for-block-volumes-).
+- Bug fixes (Issues #1, 2, 6, 7, and 8 mentioned in the known issues section of [v2.1.1](v2.1.1.md) are fixed in `v2.2.0` release)
+  - Fixed recreating static file share volume with the same PV name.
+  - Fixed metadata syncer to disallow deleting volume physically from the datastore when PV is deleted by user.
+  - Fixed VCP to CSI Migration to support datastore path in the volume handle. Refer [issue-628](https://github.com/kubernetes-sigs/vsphere-csi-driver/issues/628#issuecomment-771258857) for more detail.
+  - Fixed CSI full sync not registering volumes and was failing with `Duplicated entity for each entity type in one cluster`.
+  - Fixed volume provisioning when empty datacenter is present in the vCenter inventory.
+
```

Comment:
Do we want to list the support for Vanilla CSI  on VMC in notable changes section?

Reply:
Yes added below

> - Added support for running vSphere CSI driver on [VMware Cloud™ on AWS](https://cloud.vmware.com/vmc-aws) (VMC). VMC will only support block volume for now. The minimum SDDC version required to run vSphere CSI driver is 1.12. Please refer to [VMC release notes](https://docs.vmware.com/en/VMware-Cloud-on-AWS/0/rn/vmc-on-aws-relnotes.html) to get more details.